docs/courseInstance.md

```diff
@@ -16,61 +16,68 @@ exampleCourse
     |   |   |   `-- ...               # files for Homework 1
     |   |   `-- hw02                  # second homework for Fa16
     |   |       `-- ...               # files for Homework 2
-    |   +-- clientFilesCourseInstance
+    |   +-- clientFilesCourseInstance # (1)!
     |   |   `-- Fa16_rules.pdf        # files for Fall 2016
     `-- Sp17
         +-- infoCourseInstance.json   # Spring 2017 configuration
         +-- assessments
         |   `-- ...                   # Spring 2017 assessments
-        +-- clientFilesCourseInstance
+        +-- clientFilesCourseInstance # (2)!
         |   `-- ...                   # files for Spring 2017
 ```
 
-- See an [example course instances directory](https://github.com/PrairieLearn/PrairieLearn/blob/master/exampleCourse/courseInstances) in PrairieLearn
+1. See [clientFiles and serverFiles](clientServerFiles.md) for information on the `clientFilesCourseInstance` directory.
 
-- See [clientFiles and serverFiles](clientServerFiles.md) for information on the `clientFilesCourseInstance` directory.
+2. See [clientFiles and serverFiles](clientServerFiles.md) for information on the `clientFilesCourseInstance` directory.
+
+See an [example `courseInstances` directory](https://github.com/PrairieLearn/PrairieLearn/blob/master/exampleCourse/courseInstances) in the PrairieLearn example course.
 
 ## `infoCourseInstance.json`
 
 This file specifies basic information about the course instance:
 
-```json
+```json title="infoCourseInstance.json"
 {
   "uuid": "62fbe2a4-8c22-471a-98fe-19e5d5da1bbe",
   "longName": "Spring 2015",
-  "allowAccess": [
-    {
-      "startDate": "2015-01-19T00:00:01",
-      "endDate": "2015-05-13T23:59:59"
-    }
-  ]
+  "allowAccess": []
 }
 ```
 
-- Example [infoCourseInstance.json](https://github.com/PrairieLearn/PrairieLearn/blob/master/exampleCourse/courseInstances/SectionA/infoCourseInstance.json)
+??? example "Example `infoCourseInstance.json`"
+
+    ```json title="infoCourseInstance.json"
+    --8<-- "exampleCourse/courseInstances/SectionA/infoCourseInstance.json"
+    ```
```

Comment:
This is actually a bad example:

- It uses crazy date ranges
- It uses `"institution": "Any"`

I'd prefer to keep the dates in the existing example and just not link to the example course at all here.